### PR TITLE
Get the change from findhafeez branch to main - caption changes

### PR DESCRIPTION
**Pull request by findhafeez**
[] fixes the caption
[] updates the file.md with changes
Description - Get the change from findhafeez branch to main - caption changes

### CODE REVIEW

File: _slides/07-findhafeez.md [hunk 1]
```diff
@@ -5,5 +5,7 @@ title: "Welcome to our slide deck!"
 
 Caption here
 
+Testing findhafeez
```
Comment (thanhha87) [inline _slides/07-findhafeez.md:8]: That is good

File: _slides/07-findhafeez.md [hunk 2]
```diff
@@ -5,5 +5,7 @@ title: "Welcome to our slide deck!"
...
+
```
Comment (thanhha87) [inline _slides/07-findhafeez.md:9]: why is this an empty line?

**Review by sandhyapabbu — APPROVED** [review]

File: _slides/07-findhafeez.md (hunk 1 above)
Comment (findhafeez) [reply _slides/07-findhafeez.md:8]: Thank you
Please approve :)

**Review by thanhha87 — APPROVED** [review]
Good work

**Pull request merged**